File: bungee/src/main/java/live/turna/phenyl/bungee/loader/PhenylBungeeLoader.java
package live.turna.phenyl.bungee.loader;

import live.turna.phenyl.bungee.BungeePhenyl;
import net.md_5.bungee.api.plugin.Plugin;

public final class PhenylBungeeLoader extends Plugin {
    BungeePhenyl plugin = new BungeePhenyl(this);

    @Override
    public void onLoad(){
        plugin.onLoad();
    }

    @Override
    public void onEnable() {
        plugin.onEnable();
    }

    @Override
    public void onDisable() {
        plugin.onDisable();
    }
}

File: bungee/src/main/java/live/turna/phenyl/bungee/loader/PhenylBungeeLoader.kt
package live.turna.phenyl.bungee.loader

import live.turna.phenyl.bungee.BungeePhenyl
import net.md_5.bungee.api.plugin.Plugin

class PhenylBungeeLoader : Plugin() {
    var plugin = BungeePhenyl(this)
    override fun onLoad() {
        plugin.onLoad()
    }

    override fun onEnable() {
        plugin.onEnable()
    }

    override fun onDisable() {
        plugin.onDisable()
    }
}